Kernel/RobSubstitution.hpp

```diff
@@ -148,7 +148,7 @@ class RobSubstitution
   struct TermSpec
   {
     /** Create a new TermSpec struct */
```

Comment:
Would we prefer to set this to some invalid value? I believe this 0 is actually a reasonable value which might not cause a crash.

Reply:
Could we live without this 0-param constructor altogether?

Reply:
Seems not - it's used in a `DHMap<VarSpec, TermSpec>` member which then requires the default constructor - unless you can think of some way around it?

Reply:
Oh, I see. Then let's not waste too much effort here atm.